pedigroup/model/order.py

```diff
@@ -0,0 +1,15 @@
+class Order:
+
+    def __init__(self, group):
+        self.group = group
+        self.empanadas = list()
+        self.totalQuantity = 0
+
+    def addEmpanadas(self, empanada, quantity):
+        if quantity < 0:
+            raise ValueError("La cantidad a añadir no puede ser negativa")
+        count: int = quantity
+        while count != 0:
+            self.empanadas.append(empanada)
+            self.totalQuantity += 1
+            count -= 1
```

Comment:
Antes que usar un while podríamos hacer un for loop y sumar la cantidad total por fuera:
```suggestion
        for i in range(1, quantity):
            self.empanadas.append(empanada)
        self.totalQuantity += quantity
```